presto-main/src/main/java/com/facebook/presto/execution/executor/PrioritizedSplitRunner.java

```diff
@@ -41,6 +42,8 @@
     // each time we run a split, run it for this length before returning to the pool
     public static final Duration SPLIT_RUN_QUANTA = new Duration(1, TimeUnit.SECONDS);
 
+    private static final ThreadMXBean THREAD_MX_BEAN = (ThreadMXBean) ManagementFactory.getThreadMXBean();
```

Comment:
we don't need to cast here as the `ManagementFactory.getThreadMXBean()` returns `ThreadMXBean` right ?